### DISCUSSION TITLE
Definitelly is impossible to install Oqtane on a Windows Server

### DISCUSSION THREAD

**Discussion by ufoloko** [oqtane/oqtane.framework#4298] [General]
For few weeks I was fighting with installation process from this link https://www.oqtane.org/blog/!/24/installing-oqtane-on-iis but definitelly is not a easy task. I have tried everything but is not working. Everytime i try a new configuration o new setting, site is trying to redirect to https version for my site, i installed a temp certificate, then a blank page is opened, nothing else happens.

I dont know if maybe there is a new manual for installation, maybe something happens from old version of oqtane ?

**Comment by leigh-pointer** [comment]
Hi @ufoloko, was this closed as fixed. I am interested to know, please?

**Comment by ufoloko** [comment]
Hi Leigh, I already was able to install oqtane, now im testing functionalities.

Thanks for ask

**Reply by leigh-pointer** [reply]
@ufoloko That is great to know.  Pleased to see that you also joined the Discord group.
